### ISSUE TITLE
Change Capitalization on the VRMS Recurring Events page

### ISSUE THREAD

**Issue by jbubar** [hackforla/VRMS#1463]
### Overview
The font used for the meeting/event names on the "Recurring Events" overview of the project page on VRMS currently allows only uppercase letters, which makes it difficult for PMs to see if there is a capitalization error unless they "edit" the event or check the event name on the HfLA events webpage. We need to change the font used for the event names to a font that allows for both uppercase and lowercase letters.

### Action Items
- [ ] Change the font used for the Event / Meeting names on the Project pages to `Source Code Pro` (which should allow for both upper and lowercase).

### Resources/Instructions
<details><summary>Screenshot of Recurring Events overview with all caps meeting name</summary>
<p>

<img width="799" height="1131" alt="Image" src="https://github.com/user-attachments/assets/f0aad640-b7fa-4cee-9be7-416f787e38ad" />

</p>
</details> 

<details><summary>Screenshot of EDIT Recurring Events with correct meeting name capitalization
</summary>
<p>

<img width="704" height="1158" alt="Image" src="https://github.com/user-attachments/assets/58963972-445b-47d5-8873-e66494a086f1" />

</p>
</details>

**Comment by trillium** [comment]
TLDR: It's the font we use
